doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -241,6 +241,7 @@ Bluetooth samples
 * Added:
 
   * The :ref:`channel_sounding_ras_reflector` sample demonstrating how to implement a Channel Sounding Reflector that exposes the Ranging Responder GATT Service.
+  * The :ref:`channel_sounding_ras_initiator` sample demonstrating simple distance measurement with Channel Sounding by setting up a Channel Sounding Initiator that acts as a Ranging Requestor GATT Client.
```

Comment:
```suggestion
  * The :ref:`channel_sounding_ras_initiator` sample demonstrating distance measurement with Channel Sounding by setting up a Channel Sounding Initiator that acts as a Ranging Requestor GATT Client.
```
You never know whether it is actually simple.

Reply:
How about "minimal" or "basic" in that case?

The point of this as I read it is that since the spec for this feature puts the distance estimation part entirely on the app side, we want to communicate that:

- this sample only shows a simple (the simplest?) way of doing it
- users will likely want to do something more complex in production

Reply:
could be "minimal" or "basic" instead